test/integration/02-user-wallet/02-invoice.spec.ts

```diff
@@ -13,20 +19,36 @@ beforeAll(async () => {
 
 describe("UserWallet - addInvoice", () => {
   it("adds a self generated invoice", async () => {
-    const request = await userWallet1.addInvoice({ value: 1000 })
+    const lnInvoice = await addInvoiceForSelf({
+      walletId: userWallet1.user.id as WalletId,
+      amount: toSats(1000),
+    })
+    if (lnInvoice instanceof Error) return lnInvoice
+    const { paymentRequest: request } = lnInvoice
```

Comment:
please add and additional validation or test for pubkey

Reply:
Would checking that it starts with `lnbc1` be enough for a validation?